### PR TITLE
Csp 2999 cloudbeat eks tests

### PR DESCRIPTION
**Pull request by ofiriro3**
- Closes : [csp#2999](https://github.com/elastic/security-team/issues/2999)

### CODE REVIEW

**Review by amirbenun — COMMENTED** [review]
Seems alright at first sight. I did not have the time to go deeper.
@yashtewari will review instead.

**Review by yashtewari — APPROVED** [review]
Looks good!

File: resources/fetchers/elb_fetcher_test.go [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+			"test_namespace",
+			[]v1.LoadBalancerIngress{
+				{
+					Hostname: "adda9cdc89b13452e92d48be46858d37-1423035038.us-east-2.elb.amazonaws.com",
+				},
+			},
+			fmt.Errorf("elb error")},
+	}
+	for _, test := range tests {
+		//Need to add services
```
[the remark this answers is not in the document]
Comment (ofiriro3) [reply resources/fetchers/elb_fetcher_test.go:150]: I'll remove the comment

File: resources/fetchers/ecr_fetcher_test.go [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+
+type ECRFetcherTestSuite struct {
+	suite.Suite
+}
+
+func TestECRFetcherTestSuite(t *testing.T) {
+	suite.Run(t, new(ECRFetcherTestSuite))
+}
+
+func (s *ECRFetcherTestSuite) SetupTest() {
```
[the remark this answers is not in the document]
Comment (ofiriro3) [reply resources/fetchers/ecr_fetcher_test.go]: I wanted to leave it empty so developers could edit it in a later stage.

I removed it

**Pull request merged**